.travis.yml

```diff
@@ -22,14 +30,4 @@ script:
 - ~/miniconda/bin/python condaci.py build ./conda
 
 notifications:
-  slack:
```

Comment:
Any reason we are disabled slack build notifications too (or is this how handled though a proper integration? Don't remember)